integration_tests/tests/s3.rs

```diff
@@ -163,6 +164,8 @@ fn test_all_the_things() {
                                   &"tests/sample-data/no_credentials");
     // generate a presigned url
     test_get_object_with_presigned_url(&region, &credentials, &test_bucket, &utf8_filename);
+    test_get_object_with_expired_presigned_url(&region, &credentials, &test_bucket, &filename);
+    test_get_object_with_expired_presigned_url(&region, &credentials, &test_bucket, &utf8_filename);
```

Comment:
@matthewkmayer here it is.

Reply:
Looks like we've got an extra call to `test_get_object_with_expired_presigned_url` now. I think we only need two: one with `&filename` and one with `&utf8_filename`. That's what I was trying to say in the previous comment. 😄